doc/examples/ex11/ex11.sh

```diff
@@ -20,67 +20,67 @@ gmt begin ex11
 	gmt grdmath 255 Y SUB = y2.nc
 	gmt grdmath 255       = c2.nc
 
-	gmt plot @cut-here_11.txt -Wthinnest,. -R-51/306/0/1071 -JX3.5i/10.5i -X2.5i -Y0.5i
+	gmt plot @cut-here_11.txt -Wthinnest,. -R-51/306/0/1071 -JX8.4c/25.2c
 
 	gmt set FONT_ANNOT_PRIMARY 12p,Helvetica-Bold
 
-	gmt grdimage x1.nc y1.nc c1.nc -JX2.5i/-2.5i -X0.5i -R0/255/0/255
+	gmt grdimage x1.nc y1.nc c1.nc -JX6c/-6c -X1.25c -R0/255/0/255
 	gmt plot -Wthinner,white,- @rays_11.txt
 	gmt text --FONT=white -F+a+f <<- END
-	128 128 -45 12p 60\217
+	128 128 -45 12p 60@.
 	102  26 -90 12p 0.4
 	204  26 -90 12p 0.8
 	10  140 180 16p G
 	END
 	echo 0 0 0 128 | gmt plot -N -Sv0.15i+s+e -Gwhite -W2p,white
```

Comment:
One more at line 34.

Reply:
Hm, I don't see it? I think I got that one.